### DISCUSSION TITLE
It is recommended to add a function similar to wins to shake the current window from left to right and automatically hide other windows, which is not as convenient as shortcut keys.

### DISCUSSION THREAD

**Discussion by DevLiuSir** [rxhanson/Rectangle#1422] [General]
It is recommended to add a function similar to wins to shake the current window from left to right and automatically hide other windows, which is not as convenient as shortcut keys.

https://github.com/rxhanson/Rectangle/assets/11488337/f55491b3-b5d5-43a9-9983-9e7d9d79d12e

**Comment by rxhanson** [comment]
This isn't something I have plans to add in myself, but would merge a good pull request for it.
